transcendence-app/src/game/game.controller.ts

```diff
@@ -46,9 +49,10 @@ export class GameController {
   constructor(
     private gameService: GameService,
     private gameQueueService: GameQueueService,
+    private readonly statsService: GameStatsService,
   ) {}
 
-  @Get('pairing-status')
+  @Get('game-pairing-status')
```

Comment:
we have `game/`, `games/` location. I think pairing status should also convey the fact that it's in the game domain.

Reply:
Well, AFAIK it will try to parse `pairing-status` as a uuid for the 'game/:gameId' endpoint.
Other than that, we could put them all in the `game/` location, and have the [XXX] `game/:gameId` and [GET] `games/` endpoints be something like [GET] `game/list` and `game/user/:gameId`. But to be honest, this pattern, as it is, matches the way we have it in users, and it's kind of cleaner. Still, I'm a but unconvinced as well. Maybe if we could find better locations for the `game/` and `games/` endpoints...
What do you think?

Reply:
If I'm not mistaken, we currently don't have a `[GET] game/:gameId` endpoint, but it should work if this endpoint is before that one in the file.

Reply:
we absolutely have it in line 124 in game.controller.ts. Okay, these two are before that, so I'll change them